### ISSUE TITLE
训练结果正常吗？

### ISSUE THREAD

**Issue by wuxiaolianggit** [chineseocr/chineseocr#293]
147/147 [==============================] - 47s 319ms/step - loss: 0.0020 - acc: 0.8673
epoch:445/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:446/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:447/3000
147/147 [==============================] - 47s 319ms/step - loss: 0.0020 - acc: 0.8673
epoch:448/3000
147/147 [==============================] - 47s 319ms/step - loss: 0.0020 - acc: 0.8673
epoch:449/3000
147/147 [==============================] - 47s 321ms/step - loss: 0.0020 - acc: 0.8673
epoch:450/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:451/3000
147/147 [==============================] - 47s 319ms/step - loss: 0.0020 - acc: 0.8673
epoch:452/3000
147/147 [==============================] - 47s 321ms/step - loss: 0.0020 - acc: 0.8673
epoch:453/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:454/3000
147/147 [==============================] - 47s 322ms/step - loss: 0.0020 - acc: 0.8673
epoch:455/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:456/3000
147/147 [==============================] - 47s 321ms/step - loss: 0.0020 - acc: 0.8673
epoch:457/3000
147/147 [==============================] - 47s 318ms/step - loss: 0.0020 - acc: 0.8673
epoch:458/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:459/3000
147/147 [==============================] - 47s 319ms/step - loss: 0.0020 - acc: 0.8673
epoch:460/3000
147/147 [==============================] - 47s 323ms/step - loss: 0.0020 - acc: 0.8673
epoch:461/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:462/3000
147/147 [==============================] - 47s 320ms/step - loss: 0.0020 - acc: 0.8673
epoch:463/3000
145/147 [============================>.] - ETA: 0s - loss: 0.0020 - acc: 0.8673 

损失值已经很小了，但是准确率还是很低，请问大神这是什么情况，该怎么调整？

**Comment by wuxiaolianggit** [comment]
@zergmk2 @ @wenlihaoyu @ @callinglove 请问大神，损失值已经很小了，但是准确率还是很低，请问大神这是什么情况，该怎么调整？

**Comment by wenlihaoyu** [comment]
![image](https://user-images.githubusercontent.com/7800267/61185791-cb214480-a68f-11e9-9e23-abe91475d0ba.png)
你数据集太小。

**Comment by wuxiaolianggit** [comment]
@wenlihaoyu 我的batchsize是128，数据量大概是20000张图片，这个数据量如果小的话，大概需要多少才可以？求大神指点

**Comment by wuxiaolianggit** [comment]
@wenlihaoyu 我只进行了数字和英文字母的分类

**Comment by wenlihaoyu** [comment]
释放模型权重进行训练，你loss都没有变化

**Comment by wuxiaolianggit** [comment]
释放模型权重进行训练和不释放模型权重进行训练进行训练，有什么区别吗？大神可以解释一下吗？

**Comment by gbolin** [comment]
@wuxiaolianggit 释放模型权重 意思是说 特征提取层那里不冻结，代码上就是 requires_grad = False
